services/core-api/app/api/mines/comments/models/mine_comment.py

```diff
@@ -0,0 +1,40 @@
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.schema import FetchedValue
+
+from app.api.utils.models_mixins import Base, AuditMixin
+from app.extensions import db
+
+from app.api.utils.include.user_info import User
+from datetime import datetime
+
+
+class MineComment(Base, AuditMixin):
+    __tablename__ = "mine_comment"
+    mine_comment_id = db.Column(db.Integer, primary_key=True, server_default=FetchedValue())
+    mine_comment_guid = db.Column(UUID(as_uuid=True), nullable=False, server_default=FetchedValue())
+    mine_guid = db.Column(db.Integer, db.ForeignKey('mine.mine_guid'))
+    mine_comment = db.Column(db.String, nullable=False)
+    deleted_ind = db.Column(db.Boolean, nullable=False, default=False)
+
+    comment_user = db.Column(db.String(60), nullable=False, default=User().get_user_username)
```

Comment:
This size restriction isn't enforced on the DB level. Should it be?

Reply:
No and I will be removing this size restriction